Packs/Kafka/Integrations/KafkaV3/KafkaV3.py

```diff
@@ -0,0 +1,769 @@
+            topic_metadata = topics[topic]
+            for metadata_partition in topic_metadata.partitions.values():
+                try:
+                    offset = self.get_offset_for_partition(topic, metadata_partition.id, offset)
+                    topic_partitions += [TopicPartition(topic=topic, partition=metadata_partition.id, offset=offset)]
+                except KafkaException as e:
+                    # Sometimes listing topics can return uninitialized partitions.
+                    # If that's the case, ignore them and continue.
+                    if 'Unknown partition' not in str(e):
+                        raise e
+
+        return topic_partitions
+
+
+''' HELPER FUNCTIONS '''
+
+
+def capture_logs(func: Callable):
+    """Capture confluent kafka logs and add them when raising exceptions.
+
+    Args:
+        func: Has to support kafka_logger and log_stream kwargs
+
+    return: the func's result
+    """
+    def wrapper(*args, **kwargs):
+        logging.raiseExceptions = False
+        log_stream = StringIO()
+        log_handler = logging.StreamHandler(stream=log_stream)
+        log_handler.setFormatter(logging.Formatter(logging.BASIC_FORMAT))
+        kafka_logger = logging.getLogger()
+        kafka_logger.addHandler(log_handler)
+        kafka_logger.setLevel(logging.DEBUG)
+        debug_log = ''
+
+        try:
+            kwargs['kafka_logger'] = kafka_logger
+            kwargs['log_stream'] = log_stream
+            result = func(*args, **kwargs)
+        except Exception as e:
+            captured_logs = log_stream.getvalue()
+            if captured_logs:
+                debug_log = f'\n{str(e)}\nDebug logs:\n{captured_logs}\n'
+            else:
+                debug_log = f'\n{str(e)}\n'
+        finally:
+            if log_stream:
+                try:
+                    logging.getLogger().removeHandler(log_handler)
+                    log_stream.close()
+                except Exception as e:
+                    debug_log = f'Kafka v3: unexpected exception when trying to remove log handler:{e}\n\n' \
+                                f'Other Exceptions:{debug_log}'
+                finally:
+                    if debug_log:
+                        raise DemistoException(debug_log)
+
+        return result
+    return wrapper
+
+
+def create_incident(message: Message, topic: str) -> dict:
+    """Create incident from kafka's message.
+
+    Args:
+        message: Kafka message to create incident from
+        topic: Message's topic
+
+    Return incident
+    """
+    message_value = message.value()
+    raw = {
+        'Topic': topic,
+        'Partition': message.partition(),
+        'Offset': message.offset(),
+        'Message': message_value.decode('utf-8')
+    }
+    incident = {
+        'name': 'Kafka {} partition:{} offset:{}'.format(topic, message.partition(), message.offset()),
+        'details': message_value.decode('utf-8'),
```

Comment:
i think we need here just the `message` and not the entire object
![image](https://user-images.githubusercontent.com/31018228/144015699-69b71a51-c422-4b51-b01b-75038d1607fa.png)

Reply:
This was your message

Reply:
not sure i follow, what i meant is that from this object, we need only the `message` attribute, no need for `id`, `user` and `date`